src/Sylius/Bundle/ShopBundle/Form/Extension/ChannelTypeExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Form\Extension;
+
+use Sylius\Bundle\ThemeBundle\Form\Type\ThemeNameChoiceType;
+use Symfony\Component\Form\AbstractTypeExtension;
+use Sylius\Bundle\ChannelBundle\Form\Type\ChannelType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+final class ChannelTypeExtension extends AbstractTypeExtension
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        if (class_exists('\Sylius\Bundle\ThemeBundle\Form\Type\ThemeNameChoiceType')) {
+            $builder->add('themeName', ThemeNameChoiceType::class, [
+                'label' => 'sylius.form.channel.theme',
```

Comment:
Should translation `sylius.form.channel.theme` be moved to UiBundle to ShopBundle?

Reply:
Then we also need to duplicate the translation. But what key would you suggest.

Reply:
I think we can just keep existing key and duplication is not necessary (Since this PR targets 2.0), so to move existing `sylius.form.channel.theme` translation key to ShopBundle.

Reply:
Fair enough. Is there an easy way to do this or do I need to touch all the 46 translation files of the admin bundle? :D